tests/suites/test_suite_ssl.data

```diff
@@ -4779,3 +4779,6 @@ conf_curve:
 
 Test configuration of groups for DHE through mbedtls_ssl_conf_groups()
 conf_group:
+
+Test accessor into timing_delay_context
```

Comment:
`mbedtls_timing_delay_context` consists of `timer, intermediate delay and final delay`. This test is about final delay accessor.

My suggestion is to clarify the test description to explicitly mention that it tests accessor to final delay. Also the name of the test function can be adapted (`timing_final_delay_accessor`).